nixos/modules/services/desktops/flatpak.nix

```diff
@@ -15,6 +15,17 @@ in {
   options = {
     services.flatpak = {
       enable = mkEnableOption "flatpak";
+
+      guiPackages = mkOption {
+        type = types.listOf types.package;
```

Comment:
Perhaps this should be internal. We might want to drop it when we finish the Nix PackageKit support.

Reply:
Sounds good to make it internal, but I'm not sure why we should drop it if there's Nix PackageKit support